packages/graphql/src/resolvers/IdentityEvents.js

```diff
@@ -114,7 +121,7 @@ export function identity({ id, ipfsHash }) {
         'avatarUrl',
         'description'
       ]),
-      ...getAttestations(id, data.attestations || []),
+      ...getAttestations(accounts, data.attestations || []),
```

Comment:
Does getAttestation need to be updated to de-dupe attestations that are identical between proxy and owner ?

Reply:
It should only look at the latest IPFS hash which shouldn't contain any duplicates

Reply:
ah right. perfect.